### PR TITLE
javascript-forms | Daniel To

### PR DESCRIPTION
**Pull request by theDanielTo**
javascript-forms1
javascript-forms2
javascript-forms3
javascript-forms4

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  email: '',
+  message: ''
+};
+
+contactForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  userData.name = contactForm.elements.name.value;
+  userData.email = contactForm.elements.email.value;
+  userData.message = contactForm.elements.message.value;
+  console.log(userData);
```
Comment (scbowler) [inline javascript-forms/form-values.js:14]: You should reset the form here

**Review by scbowler — CHANGES_REQUESTED** [review]
One change needed

File: javascript-forms/form-values.js (hunk 1 above)
Comment (theDanielTo) [reply javascript-forms/form-values.js:14]: oops i missed that. changes were committed

**Review by scbowler — APPROVED** [review]
Great job! 👍

**Pull request merged**